components/Accordion/src/accordion.css

```diff
@@ -0,0 +1,122 @@
+/* accordion general styling */
+.denhaag-accordion {
+  font-family: var(--denhaag-accordion-font-family), var(--denhaag-typography-sans-serif-font-family);
```

Comment:
Fallback need to be defined slightly differently:

`var(--a), var(--b)` → `var(--a, var(--b))`

Reply:
Updated in commit: https://github.com/nl-design-system/denhaag/pull/600/commits/1ae898c3f293a8c5c8e5862c65c9f3ec4c1e8d25